test/server/api/LinkStatsRoute.test.ts

```diff
@@ -0,0 +1,49 @@
+import request from 'supertest'
+import { container } from '../../../src/server/util/inversify'
+import { DependencyIds } from '../../../src/server/constants'
+import { OtpRepositoryMock } from '../mocks/repositories/OtpRepository'
+import { OtpRepositoryInterface } from '../../../src/server/repositories/interfaces/OtpRepositoryInterface'
+import { Cryptography } from '../../../src/server/services/cryptography'
+import CryptographyMock from '../mocks/services/cryptography'
+import { UserRepositoryInterface } from '../../../src/server/repositories/interfaces/UserRepositoryInterface'
+import { MockUserRepository } from '../mocks/repositories/UserRepository'
+import { LinkStatisticsServiceMock } from '../mocks/services/LinkStatisticsService'
+import { LinkStatisticsServiceInterface } from '../../../src/server/services/interfaces/LinkStatisticsServiceInterface'
+
+// Binds mockups before binding default
+container
+  .bind<OtpRepositoryInterface>(DependencyIds.otpRepository)
+  .to(OtpRepositoryMock)
+container.bind<Cryptography>(DependencyIds.cryptography).to(CryptographyMock)
+container
+  .bind<UserRepositoryInterface>(DependencyIds.userRepository)
+  .to(MockUserRepository)
+container
+  .bind<LinkStatisticsServiceInterface>(DependencyIds.linkStatisticsService)
+  .to(LinkStatisticsServiceMock)
+
+// Importing setup app
+import app from './setup'
+
+describe('GET: /api/link-stats', () => {
+  test('positive test: Should get statistics of mocked link', async (done) => {
```

Comment:
```suggestion
  it('should get statistics of mocked link', async (done) => {
```
The goal here is to let it read like a spec document. the messages that are passed to `it/test` are also formed as part of the test report; if you follow convention, it should look something like this in the CLI:
![image](https://user-images.githubusercontent.com/10572368/94400678-1728ae00-019c-11eb-94ec-7e878d7f9dca.png)